tutorials/Python/12_AnalysingMD/AnalysingMD.ipynb

```diff
@@ -4,7 +4,17 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# 11. Analysing MD data"
+    "# 12. Analysing MD data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Important note:\n",
```

Comment:
@RMeli thoughts on this? sufficient? too little?

Reply:
I'd remove the `:`, otherwise I think it's really clear.